lib/jbuilder/jbuilder_template.rb

```diff
@@ -259,7 +267,8 @@ def _render_explicit_partial(name_or_options, locals = {})
     else
       # partial! 'name', locals: {foo: 'bar'}
       if locals.one? && (locals.keys.first == :locals)
-        options = locals.merge(partial: name_or_options)
+        locals[:partial] = name_or_options
```

Comment:
Mutating the argument. We should avoid it.

Reply:
This one I don't understand. `locals` is already being mutated in this method with the current logic. Further down below, a `locals.delete(:as)` is run, and this can happen directly against the `locals` argument.

Reply:
Ah ok. This was one of the reasons why I thought there were no issues mutating the options since it was technically being done already. Didn't consider that this may not have been deliberate. 

I'll try to address this, too.